src/containers/Network/UpgradeStatus.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import axios from 'axios'
+import BarChartVersion from './BarChartVersion'
+import NetworkTabs from './NetworkTabs'
+import Streams from '../shared/components/Streams'
+import Hexagons from './Hexagons'
+import { localizeNumber } from '../shared/utils'
+import { useLanguage } from '../shared/hooks'
+import Log from '../shared/log'
+
+export const UpgradeStatus = () => {
+  const [vList, setVList] = useState<any>([{}])
+  const [validations, setValidations] = useState([])
+  const [unlCount, setUnlCount] = useState(0)
+  const { t } = useTranslation()
+  const language = useLanguage()
+
+  const aggregateData = (validators: any[]) => {
+    if (!validators) {
+      return []
+    }
+    let total = 0
+    const tempData: any[] = []
+    validators.reduce((aggregate, current) => {
+      if (!aggregate[current.server_version]) {
+        // eslint-disable-next-line no-param-reassign
```

Comment:
For these eslint comments can you make sure to add comments explaining why it's necessary? (Or find a way to avoid them?)

I often find when I try explaining why I need to do them, I end up finding a way to avoid it :P